theory.md

```diff
@@ -8,6 +8,10 @@ a. Single regex that matches either of these:
     
     antelopes rock out
 
+
+    ====== ANS ======
+    antelopes?\srocks?\sout
```

Comment:
Nice one, it matches both full sentences. If You only want to match `antelope` or `antelopes` You could easily do `antelopes?`